src/server/api/routes/users/v2/event.js

```diff
@@ -1,20 +1,41 @@
 const { Router } = require('express');
 const {
   walletUserCollection,
-  FieldValue,
+  Timestamp,
 } = require('../../../../modules/firebase');
 const { authenticateV2Login } = require('../../../middleware/auth');
 const { setPrivateCacheHeader } = require('../../../middleware/cache');
 const { publisher, PUBSUB_TOPIC_MISC } = require('../../../../modules/pubsub');
 
 const router = Router();
 
+function isValidTs(ts, lastSeenTs) {
+  // eslint-disable-next-line no-restricted-globals
+  if (isNaN(ts) || ts % 1000 !== 0) {
+    return false;
+  }
+  if (ts < lastSeenTs) {
+    return false;
+  }
+  if (ts > Date.now() + 5 * 60 * 1000) {
+    return false;
+  }
+  return true;
```

Comment:
I think these condition can be converted into a multi line boolean, in which we can just return its value
i.e.
```
return (!(isNaN(ts) || ts % 1000 !== 0)
  && (ts > lastSeenTs)
  && (ts < Date.now() + 5 * 60 * 1000)
```